features/user_registration.feature

```diff
@@ -0,0 +1,27 @@
+Feature: As a user in order to use the application
+    I should be able to sign up for account
+    I should be able to log in
+    I should be able to log out
+
+    Scenario: Register User
+      Given I'm on the "home" page
+      And I click on "Sign up"
+      Then I should be on the "Sign up" page
```

Comment:
You should move away from having assertions in the middle of a scenario. In this case, If the next step fails you are clearly not on this page. Meaning that the next step is a kind on implicit assertion.